### DISCUSSION TITLE
Pure out of curiosity, is it possible to use CrazyFlie drones outside?

### DISCUSSION THREAD

**Discussion by Bartvt21** [bitcraze/discussions#1415] [General]
Pure out of curiosity, is it possible to use CrazyFlie drones outside? 

Do external conditions like wind and other forms of noice be filtered out?

**Comment by gemenerik** [comment]
The Crazyflie is designed for indoor flight. As a flying PCB, it lacks weatherproofing by default, with particular vulnerability to water. While its controllers are capable of managing noise, they have not been optimized for windy conditions. However, due to its open-source nature, adjustments can certainly be made to address this limitation.
